src/about.html

```diff
@@ -0,0 +1,411 @@
+                <a
+                  class="nav-link fw-semibold text hover-effect"
+                  aria-current="page"
+                  href="../home.html"
+                >
+                  Home
+                </a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link text hover-effect" href="./about.html">
+                  About
+                </a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link text hover-effect scroll" href="#Teachers">
+                  Our Teachers
+                </a>
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link text hover-effect scroll" href="#Courses">
+                  Courses
+                </a>
+              </li>
+              <li class="nav-item me-5 ms-2">
+                <button
+                  type="button"
+                  class="navBtn main-color px-5 py-3 fw-bold"
+                >
+                  Join us
+                </button>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="nav-menu hidden hidden-m hidden-lg d-md-none pt-5 bg-light">
+      <ul class="navbar-nav nav_content">
+        <li class="nav-item me-3">
+          <a
+            class="nav-link fw-semibold fs-1 text hover-effect"
+            aria-current="page"
+            href="./home.html"
+          >
+            Home
+          </a>
+          <div class="border-bottom border-1 mb-3 border-danger"></div>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link text fs-2 mb-1 hover-effect" href="./about.html">
+            About
+          </a>
+          <div class="border-bottom border-1 mb-3 border-danger"></div>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link text fs-2 mb-1 hover-effect" href="#">
+            Our Teachers
+          </a>
+          <div class="border-bottom border-1 mb-3 border-danger"></div>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link text fs-2 mb-1 hover-effect" href="#">Courses</a>
+          <div class="border-bottom border-1 mb-3 border-danger"></div>
+        </li>
+        <li class="nav-item me-3 mt-5 ms-2">
+          <button
+            type="button"
+            class="btn btn-outline-danger border border-5 border-danger px-5 fw-bold"
+          >
+            Join us
+          </button>
+        </li>
+      </ul>
+    </div>
+    <section class="container-fluid mt-2 pt-3 headline-about">
+      <div class="container text-center mt-lg-5">
+        <div class="col_size container">
+          <div
+            class="pt-3 p-3 text container head-container d-flex flex-column justify-content-center align-items-center"
+          >
+            <h1 class="head-primary text-center">
+              MAMA'S KITCHEN ONLINE COOKING SCHOOL
+            </h1>
+            <div
```

Comment:
- [ ] Kindly consider making sure that this title looks exactly the same as the one provided in the design to make sure that your design is of a high standard and also remember to add the small heading with your own content above the main heading 
- your design 👇 
<img width="374" alt="Screen Shot 2023-05-23 at 17 04 55" src="https://github.com/Yonashenok/first-capstone-project/assets/60146030/43097627-1648-4928-a532-99b14006aeff">

- given design 👇 
<img width="453" alt="Screen Shot 2023-05-23 at 17 05 27" src="https://github.com/Yonashenok/first-capstone-project/assets/60146030/9c2a5b55-6534-4639-bb03-b15115447bdc">


☠️ kindly consider doing so on both screen size